README.md

```diff
@@ -153,6 +153,7 @@ Before implementing a feature, please open an Issue first, so we can be sure tha
 - Nic Desjardins, @nicdesjardins
 - Vladislav Kuznecov, @fizvlad
 - Oleg V., @neooleg
+- Manuel Jesús Aguilera Castro, @manueljaguilera
```

Comment:
Could you please also update CHANGELOG.md and settings3.html?  

in settings3.html you need to add something like this:  `<option value="fr">Français</option>`